modules/test/playwright/tests/export-import-web/export.spec.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {productMenuPageTest} from '../../fixtures/productMenuPageTest';
+import {uiElementsPageTest} from '../../fixtures/uiElementsTest';
+import {companyExportImportPageTest} from './fixtures/companyExportImportPagesTest';
+import {exportImportPagesTest} from './fixtures/exportImportPagesTest';
+
+export const test = mergeTests(
+	applicationsMenuPageTest,
+	companyExportImportPageTest,
+	exportImportPagesTest,
+	featureFlagsTest({
+		'LPD-35914': {enabled: true, system: true},
+	}),
+	loginTest(),
+	productMenuPageTest,
+	uiElementsPageTest
+);
+
+test(
+	'can see the same Deletions label at both the instance and company levels',
```

Comment:
Hey @magjed4289!
Could you adapt the test to the new structure of  `instance.export.spec.ts`?

There is already a method called `can see corresponding elements at instance level` in which this may fit